server/public/shared/markdown/inspect.go

```diff
@@ -3,9 +3,18 @@
 
 package markdown
 
+var (
+	// Assuming 64k maxSize of a post which can be stored in DB.
+	// Allow scanning upto twice(arbitrary value) the post size.
```

Comment:
If we are not using just the maxSize of a post, could we also add a comment [here](https://github.com/mattermost/mattermost-server/blob/abcfa8cce8c39529ecb61b2094ab15614adbda00/server/channels/store/sqlstore/post_store.go#L2555) so that any change on the store side is also reflected here? Just so that users are aware there is another limitation based on the maxPostSize limitation. 

Also, any reason this is not a const?

Lastly, let's also add a test to verify the behavior. Thanks.